### PR TITLE
[wip] vSphere: add cns volume delete to destroy

### PR DESCRIPTION
**Pull request by jcpowermac**

### CODE REVIEW

**Comment by jcpowermac** [comment]
@gnufied is this what you were thinking?

**Comment by gnufied** [comment]
@jcpowermac yes that is what I was thinking. It would allow us to reuse stuff from CI and then developers can use this for use-throw environments..

File: cmd/openshift-install/destroy.go [hunk 1]
```diff
@@ -58,11 +62,13 @@ func newDestroyClusterCmd() *cobra.Command {
 			logrus.Infof("Uninstallation complete!")
 		},
 	}
+	destroyCmd.Flags().BoolVar(&deleteVolumes, "delete-volumes", false, "Delete CNS volumes on destroy.")
```
Comment (jcpowermac) [inline cmd/openshift-install/destroy.go:65]: @patrickdillon installer doesn't have flags usually, any recommendation for alternative here?

**Pull request closed without merging**